src/perfume.ts

```diff
@@ -174,42 +188,51 @@ export default class Perfume {
   }
 
   /**
-   * Sends the User timing measure to Google Analytics.
-   * ga('send', 'timing', [timingCategory], [timingVar], [timingValue])
-   * timingCategory: metricName
-   * timingVar: googleAnalytics.timingVar
-   * timingValue: The value of duration rounded to the nearest integer
+   * Records the User timing measure by sending it to:
+   * - any custom analytics tracker
+   * - built-in Google Analytics if enabled if/when possible.
    */
   sendTiming(metricName: string, duration: number): void {
     // Doesn't send timing when page is hidden
     if (this.isHidden) {
       return;
     }
 
-    // Send metric to custom Analytics service,
-    // the default choice is Google Analytics
-    if (this.config.analyticsLogger) {
-      this.config.analyticsLogger(metricName, duration);
-    }
+    const metric = new Metric(metricName, duration);
 
-    if (!this.config.googleAnalytics.enable) {
-      return;
-    }
-    if (!window.ga) {
-      this.logWarn(
-        this.config.logPrefix,
-        'Google Analytics has not been loaded',
-      );
-      return;
-    }
-    const durationInteger = Math.round(duration);
-    window.ga(
-      'send',
-      'timing',
-      metricName,
-      this.config.googleAnalytics.timingVar,
-      durationInteger,
-    );
+    this.analyticsTrackers.forEach(tracker => {
+      if (tracker.canSend()) {
+        tracker.send(metric);
+      } else {
+        tracker.metricQueue = tracker.metricQueue || [];
+        tracker.metricQueue.push(metric);
+        this.logWarn(
+          this.config.logPrefix,
+          `${tracker.name} is not ready; metric will be ` +
+            'queued until window.load and tried then.',
+        );
+      }
+    });
+  }
+
+  /**
+   * window.load handler.
+   * Tries to send any queued metrics to trackers.
+   */
+  onWindowLoad(): void {
```

Comment:
We use the prefix `did` for listener callback from the `document` or `window`, can you please change this to `didWindowLoad`.

Reply:
Done.